calcom/cal.com
# fix booking detail page for cancelled events

## What does this PR do?

Fixes booking detail page for cancelled events.

Before: 
![Screenshot 2022-05-27 at 14 52 49](https://user-images.githubusercontent.com/30310907/170702901-c0285003-2b2f-4418-939b-f813a9e92978.png)


After:
<img width="763" alt="Screenshot 2022-05-27 at 14 44 51" src="https://user-images.githubusercontent.com/30310907/170702143-42f875fb-6b1f-4330-bfad-6d305a46d784.png">


**Environment**: Staging(main branch)

## Type of change




## How should this be tested?

- Go to Bookings
- Click on a cancelled booking
- You should see the view from the screenshot above

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
